addons/project/views/project_views.xml

```diff
@@ -1709,8 +1709,16 @@
             <field name="view_id" ref="view_task_all_calendar"/>
         </record>
 
+        <record id="action_menu_view_all_task" model="ir.actions.server">
+            <field name="name">menu View all task</field>
+            <field name="model_id" ref="project.model_project_task"/>
+            <field name="state">code</field>
+            <field name="code">
+                action = model.action_menu_view_all_task()
+            </field>
+        </record>
         <menuitem name="My Tasks" id="menu_project_management" parent="menu_main_pm"
-            action="action_view_all_task" sequence="2" groups="base.group_no_one,group_project_user"/>
+            action="action_menu_view_all_task" sequence="2" groups="base.group_no_one,group_project_user"/>
```

Comment:
We could rename the action name to say it is for My Tasks only because now we have All Tasks menu. (That's because of this new menu, you have the conflict because the actions have been renamed) 😇